public/app/core/utils/kbn.js

```diff
@@ -241,6 +241,32 @@ function($, _) {
     return formatted;
   };
 
+  kbn.toFixedDotted = function(value, decimals) {
+    if (value === null) {
+      return "";
+    }
+
+    var factor = decimals ? Math.pow(10, Math.max(0, decimals)) : 1;
+    var formatted = (Math.round(value * factor) / factor).toLocaleString('de-DE');
```

Comment:
de-DE was one of the locales that uses this particular notation. If you're aware of a more generic way, feel free to suggest.